activities/TamTamMicro.activity/piano.js

```diff
@@ -0,0 +1,81 @@
+enyo.kind({
+	name: "TamTam.Piano",
+    components: [
```

Comment:
Please don't mix indentation, should be TAB

Reply:
3e59733